.env.example

```diff
@@ -1,2 +1,3 @@
 WALLET_KEY= # the private key of the wallet
-ENCRYPTION_KEY= # a second random 32 bytes encryption key for local db encryption
\ No newline at end of file
+ENCRYPTION_KEY= # a second random 32 bytes encryption key for local db encryption
+OPENAI_API_KEY= # the API key for the OpenAI API
```

Comment:
add a comment above this var saying it's only necessary for the GPT agent example